alsoHeresTheInterviewQuestionIHad/alsoHeresTheInterviewQuestionIHad.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const streetArray = fs.readFileSync("data.txt", "utf-8").split(",");
+
+const streetArraySorted = streetArray.sort(function (a, b) {
+    return a - b;
+});
+
+let numberOfBuildings = streetArray.length;
+
+let tallestBuildingSoFar = 0;
+let tallestBuilding = Math.max(...streetArray);
+
+let bannerOneHeight = 0;
+let bannerOneWidth = 0;
+let bannerOneArea = tallestBuilding * numberOfBuildings;
+let bannerTwoHeight = 0;
+let bannerTwoWidth = 0;
+let bannerTwoArea = 0;
+
+function findBestArea() {

```

Comment:
I'd expect this function to accept the array of building heights, that way we can reuse this function